qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -64,14 +68,15 @@ def __init__(
         """Initialize pulse schedule to signals converter.
 
         Args:
-            dt: Length of the samples. This is required by the converter as pulse schedule are
-                specified in units of dt and typically do not carry the value of dt with them.
-            carriers: A dict of analog carrier frequencies. The keys are the names of the channels
+            dt: Length of the samples. This is required by the converter as pulse
```

Comment:
These two code blocks are similarly changing some documentation I had changed previously. Maybe some issue with a previous attempt at merging `main` into this branch?